chaoscenter/cypress.config.js

```diff
@@ -2,7 +2,11 @@ const { defineConfig } = require("cypress");
 
 module.exports = defineConfig({
   e2e: {
-    baseUrl: 'http://192.168.49.2:30829',
+    baseUrl: "http://localhost:3000", // default value
+    setupNodeEvents(on, config) {
+      config.env.baseUrl = process.env.CYPRESS_baseUrl || config.env.baseUrl;
+      return config;
+    },
```

Comment:
I think we can directly use this env - https://docs.cypress.io/guides/guides/environment-variables#Overriding-Configuration